tests/test_geointerface.py

```diff
@@ -71,6 +74,61 @@ def test_geointerface(self):
         self.assertIsInstance(shape, MultiPolygonAdapter)
         self.assertEqual(len(shape.geoms), 1)
 
+    def test_geointerface(self):
```

Comment:
I copied the test above, and adapted it to use `shape` instead of `asShape`, as there didn't seem a dedicated test for `shape` for all different geometry types.